### ISSUE TITLE
in add action, list the annotation created instead of latest one

### ISSUE THREAD

**Issue by mrclay** [teamwebgalli/galliComments#2]
When multiple people are commenting, you'll have a race condition. You might end up returning another user's comment instead of the added one.

**Comment by teamwebgalli** [comment]
yup Steve, we are aware of it. planned it for next release. we just need to pass the owner_guid in elgg_list_annotations(). this will return the users last comment.
